### DISCUSSION TITLE
Dreame L10 Pro - UART Login issue

### DISCUSSION THREAD

**Discussion by Yersi88** [Hypfer/Valetudo#1675] [Q&A / Support] [answered]
Hello, 

I followed the steps here: https://valetudo.cloud/pages/installation/dreame.html 

I connected the UART and I got the 'root password changed' message, however, when I press the 'Reset' button near the dustbin. All I get in console is this line: 
`cat: can't open '/etc/release': No such file or directory`

There is no login prompt, am I doing something wrong here? 

Many thanks.

**Comment by Hypfer** [comment, marked as answer]
For support regarding this issue, please head over to one of the support channels that you can find at the bottom of the readme of this repo: https://github.com/Hypfer/Valetudo#join-the-discussion
